crates/forge_loader/src/manifest.rs

```diff
@@ -86,11 +157,28 @@ pub struct ForgeModules<'a> {
     scheduled_triggers: Vec<ScheduledTrigger<'a>>,
     #[serde(rename = "consumer", default, borrow)]
     pub consumers: Vec<Consumer<'a>>,
+    #[serde(rename = "compass:dataProvider", default, borrow)]
```

Comment:
<!--
 Copyright 2024 Joshua Wong.
 SPDX-License-Identifier: Apache-2.0
-->

The following compass modules are missing:

- [`compass:componentPage`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/compass-component-page/#properties)
- [`compass:globalPage`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/compass-global-page/#properties)
- [`compass:teamPage`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/compass-team-page/#properties)
- [`compass:adminPage`](https://developer.atlassian.com/platform/forge/manifest-reference/modules/compass-admin-page/#properties)

Reply:
Done